cedars/app/db.py

```diff
@@ -875,12 +905,15 @@ def update_project_name(new_name):
     mongo.db["INFO"].update_one({}, {"$set": {"project": new_name}})
 
 
-def mark_annotation_reviewed(annotation_id):
```

Comment:
this parameter `skip_after_event` should not exist here as the function `mark_annotation_reviewed` should do one thing and that is mark the annotation reviewed. This variable is also not getting used.